### PR TITLE
Generic delete() function requested in the reviews

### PR DESCRIPTION
**Pull request by cemal-efe-g**
I tried to update all delete() calls I could but I might have missed one or made a mistake, as I am not familiar with the code, if you could please check the implementations.

### CODE REVIEW

File: dozer/db.py [hunk 1]
```diff
@@ -136,24 +136,16 @@ def __repr__(self):
...
-        """Deletes by two criteria"""
-        async with Pool.acquire() as conn:
-            statement = f"""
-            DELETE FROM  {cls.__tablename__}
-            WHERE {data_column} = {data} AND {data_column_two} = {data_two};
-            """
+            WHERE {data_tuple_list[0][0]} = {data_tuple_list[0][1]}"""
+            if len(data_tuple_list) > 1:
+                for i in range(1, len(data_tuple_list)):
+                    statement += f" AND {data_tuple_list[i][0]} = {data_tuple_list[i][1]}"
```
Comment (tweirtx) [inline dozer/db.py]: First of all, I want to thank you for taking the time to work on this.

Second, unfortunately this is vulnerable to SQL injection. Using asyncpg query parameters would eliminate the vulnerability.

**Review by tweirtx — CHANGES_REQUESTED** [review]
Fix the SQL injection vuln and I'll definitely merge this in. Thanks!

File: dozer/db.py (hunk 1 above)
Comment (cemal-efe-g) [reply dozer/db.py]: Oops, my bad right there. I am quite new to SQL, thanks for showing that out.

**Comment by tweirtx** [comment]
Thanks!

**Pull request merged**